src/node-lib/curriculum-api-2023/queries/lessonDownloadsCanonical/lessonDownloadsCanonical.query.ts

```diff
@@ -1,6 +1,8 @@
 import OakError from "../../../../errors/OakError";
 import { Sdk } from "../../sdk";
 import { lessonPathwaySchema } from "../../shared.schema";
+import { constructDownloadsArray } from "../lessonDownloads/downloadUtils";
```

Comment:
Can we remove this downloads canonical query, and do this in downloads query, so we only have one to maintain ?  Like the lesson overview

Reply:
Refactored ✅